src/messages/create/publish.ts

```diff
@@ -57,6 +57,7 @@ export async function PutContentToStorageEngine<T>(configuration: PutConfigurati
         }
     }
     if (!configuration.inlineRequested) {
+        configuration.message.item_content = null;
```

Comment:
May clash with [this](https://github.com/aleph-im/aleph-sdk-ts/pull/21/files/4cde0d92323852d0623c45567d45e5124af11553#diff-639bcfb0803e12ae1b4027e3e41e4960a5477e048dc15bd2acc14cd1b1395743), `null` seems cleaner than undefined

Reply:
As this field have to be removed when sending this type of content, this syntax also made sense:
`delete configuration.message.item_content;`

but behind it's just putting the field to `null`

Reply:
Updated: https://github.com/aleph-im/aleph-sdk-ts/pull/48/commits/62b0f9c2b269a701897bf87a12311f37812fac33
Following the aleph-client implementation, removing this field from the message

Reply:
Update: https://github.com/aleph-im/aleph-sdk-ts/pull/48/commits/90c90c10cf90a43f6f8c97dbb73f0665c95c20f6
This method cause a 422 error, so I follow [Mike implem](https://github.com/aleph-im/aleph-sdk-ts/pull/21/files/4cde0d92323852d0623c45567d45e5124af11553#diff-639bcfb0803e12ae1b4027e3e41e4960a5477e048dc15bd2acc14cd1b1395743)